app/views/design-system/rsd-design-system/index.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layouts/content-with-sidenav.html" %}
+
+{% set pageName = "Regional Services Division design system" %}
+{% set pageDescription = "Guidelines for designing and building Regional Services Division products. " %}
+{% set selectedNav = "Design system" %}
+{% set setSelectedSideNav = "" %}
+{% set parentSection = "rsd-design-system"%}
+{% set isChildSection = "true"%}
+{% set lastUpdated = "June 2024 "%}
+{% set backlog_issue_id = "" %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+<p>You can use the Regional Services Division (RSD) design system to help you design and build <a href="https://design-histories.education.gov.uk/team/regional-services-division" target="_blank" rel="noopener noreferrer">RSD products</a>.
+<p>This will help ensure we create a consistent experience for users who use multiple products.</p>
+<h2>GOV.UK and DfE Design System</h2>
+<p>If there's anything that does not exist in the RSD design system, you can use the <a href="https://design-system.service.gov.uk/get-started/" target="_blank" rel="noopener noreferrer">GOV.UK Design System</a> and <a href="/design-system/">DfE Design System</a>.</p>
```

Comment:
Don't need the trailing / here. Just use "/design-system"

Reply:
Thanks Andy the trailing / is removed.